tqsdk/connect.py

```
@@ -255,6 +256,9 @@ async def _send_handler(self, send_chan, client):
                         if time.time() - first_time < 1:
                             warnings.warn(f"1s 内订阅请求次数超过 {self._subscribed_per_seconds} 次，订阅多合约时推荐使用 api.get_quote_list 方法。", stacklevel=3)
                     self._subscribed_queue.put(time.time())
+                if pack.get("aid") == "ins_query":
+                    if len(pack.get("query", "")) > self._query_max_length:
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)